spring-5-security/src/test/java/com/baeldung/xss/PersonControllerUnitTest.java

```diff
@@ -0,0 +1,65 @@
+package com.baeldung.xss;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.*;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+import java.io.IOException;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class PersonControllerUnitTest {
+
+    @LocalServerPort
+    int randomServerPort;
+
+    private static String createPersonUrl;
+    private static RestTemplate restTemplate;
+    private static HttpHeaders headers;
+    private static UriComponentsBuilder builder;
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    private static JSONObject personJsonObject;
```

Comment:
It looks like these `private static` variables should actually be local variables in the test. They're most definitely not global variables for the test fixture!

Reply:
Done.